tests/docker_for_galaxy.py

```diff
@@ -0,0 +1,48 @@
+from collections import namedtuple
+
+import docker
+import pytest
+from bioblend.galaxy import GalaxyInstance
+
+from ephemeris.sleep import galaxy_wait
+
+
+# It needs to work well with dev. Alternatively we can pin this to 'master' or another stable branch.
+# Preferably a branch that updates with each stable release
+GALAXY_IMAGE = "bgruening/galaxy-stable:dev"
```

Comment:
That's fine for now, but we can also parameterize the start_container fixture to control which image to start.
Or the simpler variant of controlling it with an ENV var `GALAXY_IMAGE = os.environ.get("EPHEMERIS_GALAXY_DOCKER_IMAGE, "bgruening/galaxy-stable:dev")`